tests/test_external_search.py

```diff
@@ -945,38 +939,49 @@ class TestSearchFilterFromLane(DatabaseTest):
     def test_make_filter_handles_collection_id(self):
         search = DummyExternalSearchIndex()
 
-        lane = Lane(
-            self._db, self._default_library, "anything", 
-        )
+        lane = self._lane("anything")
         collection_ids = [x.id for x in lane.library.collections]
         filter = search.make_filter(
             collection_ids,
-            lane.media, lane.languages, lane.exclude_languages,
-            lane.fiction, list(lane.audiences), lane.age_range,
+            lane.media, lane.languages,
+            lane.fiction, list(lane.audiences), lane.target_age,
             lane.genre_ids,
         )
-        collection_filter, medium_filter = filter['and']
+        [collection_filter] = filter['and']
```

Comment:
why did this change?

Reply:
Previously, if you created a generic Lane it had a restriction to the ebook medium by default. That's no longer the case -- there's no medium restriction unless you specify one.